src/vendors/theme/assets/styles/_misc.scss

```
@@ -40,17 +40,24 @@ body {
 }
 
 .main-panel {
+  @include display-flex();
+  @include flex-direction(column);
```

Side note: We should probably stop using these mixins since we only support the latest version.

I suppose this is equivalent

```suggestion
  display: flex;
  flex-direction: column;
```

---

Not sure what these mixins do, just moved the `@include`s on top, will check tomorrow.

---

I think they're part of Compass, which added prefixes if I remember correctly. Their docs mention the various flexbox implementations that existed before flexbox was finalized.